pkg/controller/servicebindingrequest/mapper.go

```diff
@@ -12,7 +21,97 @@ var (
 
 // sbrRequestMapper is the handler.Mapper interface implementation. It should influence the
 // enqueue process considering the resources informed.
-type sbrRequestMapper struct{}
+type sbrRequestMapper struct {
+	client     dynamic.Interface
+	restMapper meta.RESTMapper
+}
+
+var serviceBindingRequestGVK = v1alpha1.SchemeGroupVersion.WithKind("ServiceBindingRequest")
+var secretGVK = corev1.SchemeGroupVersion.WithKind("Secret")
+
+// isServiceBindingRequest checks whether the given obj is a Service Binding Request through GVK
+// comparison.
+func isServiceBindingRequest(obj runtime.Object) bool {
+	return obj.GetObjectKind().GroupVersionKind() == serviceBindingRequestGVK
+}
+
+// isSecret checks whether the given obj is a Secret through GVK comparison.
+func isSecret(obj runtime.Object) bool {
```

Comment:
We should also add a check to see if an object is `configMap`

Reply:
In this particular context only `corev1.Secret` resources should be checked since it might appear in the `.status.secret` field of a SBR; I can't see whether the comparison you're mentioning fits, could you clarify?

Reply:
Oh Okay!